src/guide/javascript.md

```diff
@@ -191,34 +207,135 @@ const kp = generateKeyPair({
+
+`Torii` class represents HTTP / WebSocket communications with Iroha. It
+will help us to communicate with each Iroha Endpoint, i.e.:
+
+- Submit transactions
+- Make queries
+- Listen for events
+- Listen for blocks stream
+- and so on
+
+To initialize `Torii`, first we need to know Iroha Torii URLs. Our Iroha
+Peer is configured to listen for API endpoints at `http://127.0.0.1:8080`
+and for telemetry endpoints at `http://127.0.0.1:8081`. Then, we need to
+provide appropriate HTTP / WebSocket adapters which `Torii` will use[^1].
+These adapters depends on the environment you are going to use
+`@iroha2/client` in.
+
+[^1]:
+    We have to pass environment-specific `ws` and `fetch`, because there is
+    no way for Iroha Client to communicate with a peer in an
+    environment-agnostic way.
+
+In Node.js `Torii` initialization will look like this:
+
+```ts
+import { Torii } from '@iroha2/client'
+import { adapter as WS } from '@iroha2/client/web-socket/node'
+
+import nodeFetch from 'node-fetch'
+
+// another alternative
+import { fetch as undiciFetch } from 'undici'
+
+const torii = new Torii({
+  apiURL: 'http://127.0.0.1:8080',
+  telemetryURL: 'http://127.0.0.1:8081',
+  ws: WS,
+  fetch: nodeFetch as typeof fetch,
+})
+```
+
+::: tip
+
+In the example above, we use
+[`node-fetch`](https://www.npmjs.com/package/node-fetch) package which
+implements [Fetch API](https://fetch.spec.whatwg.org/#fetch-method) in
+Node.js. However, you can use
+[`undici`](https://undici.nodejs.org/#/?id=undicifetchinput-init-promise)
+as well.
+
+:::
+
+::: info
+
+`fetch: nodeFetch as typeof fetch` type assertion is acceptable here for a
+reason. `Torii` expects the "classic", native `fetch` function which is
+available natively in Browser. However, both `node-fetch` and `undici`
+don't provide `fetch` that is 100% compatible with the native one. Since
+`Torii` doesn't rely on those corner-features that are partially provided
+by `node-fetch` and `undici`, it's fine to ignore the TypeScript error
+here.
+
+:::
+
+And here is the sample of `Torii` initialization in Browser:
+
+```ts
+import { Torii } from '@iroha2/client'
+import { adapter as WS } from '@iroha2/client/web-socket/native'
+
+const torii = new Torii({
+  apiURL: 'http://127.0.0.1:8080',
+  telemetryURL: 'http://127.0.0.1:8081',
+  ws: WS,
+  // passing globally available `fetch`
+  fetch: fetch.bind(window),
+})
+```
+
+::: info NOTE
+
+We make `fetch.bind(window)` to avoid
+`TypeError: "'fetch' called on an object that does not implement interface Window."`.
+
+:::
+
+Great! Now we have `signer` and `torii`. Finally, we could create a
+`Client`:
+
+```ts
+import { Client, Signer, Torii } from '@iroha2/client'
+
+// --snip--
+declare const torii: Torii
+declare const signer: Signer
+
+const client = new Client({ torii, signer })
 ```
 
+`Client` provides a couple of tiny, but useful utilities for transactions
+and queries. Other things could be done without it directly via `Torii`.
```

Comment:
what are `other things`?

Reply:
Direct and "raw" communications with endpoints. For example, `Client.submitExecutable(executable)` accepts executable, wraps it into a transaction payload, makes signature from it, wraps it into `VersionedTransaction` and finally calls `torii.transaction(tx)`. Same for `Client.requestWithQueryBox()`. And by "other things" I mean that user could use other endpoints directly via `Torii`.

Reply:
see my suggestion: https://github.com/hyperledger/iroha-2-docs/pull/177#discussion_r973983818

Reply:
Fixed